### PR TITLE
Add Event Signup UI

### PR DESCRIPTION
**Pull request by nikhitashanker**
Add UI for event signup.

Screenshot 2020-07-23 at 5 09 17 PM

Screenshot 2020-07-23 at 4 13 21 PM

### CODE REVIEW

File: project/src/main/webapp/style.css [hunk 1]
```diff
@@ -0,0 +1,4 @@
+.submit {
```
Comment (dash102) [inline project/src/main/webapp/style.css:1]: Make a separate CSS file for this page

File: project/src/main/webapp/signup.html [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+    />
+    <script
+      src="https://stackpath.bootstrapcdn.com/bootstrap/4.5.0/js/bootstrap.min.js"
+      integrity="sha384-OgVRvuATP1z7JjHLkuOU7Xw704+h835Lr+6QL9UvYjZE3Ipu6Tp75j7Bh/kR0JKI"
+      crossorigin="anonymous"
+    ></script>
+  </head>
+  <body class>
+    <div class="container">
+      <div class="card" style="width: 18rem;">
```
Comment (dash102) [inline project/src/main/webapp/signup.html:25]: move out of in-line styling

Comment (nikhitashanker) [reply project/src/main/webapp/signup.html]: Done.

File: project/src/main/webapp/style.css (hunk 1 above)
Comment (nikhitashanker) [reply project/src/main/webapp/style.css]: Done.

**Pull request merged**